mbed_app.json

```diff
@@ -430,7 +416,10 @@
             "storage.storage_type"                      : "TDB_INTERNAL"
         },
         "GR_LYCHEE": {
-            "target.macros_add"                         : ["MBEDTLS_USER_CONFIG_FILE=\"mbedTLSConfig_mbedOS.h\""],
+            "events.shared-eventsize"                   : 2048,
```

Comment:
no need to repeat the events here. Its already on top-section.

Reply:
Done.